src/Paystack.php

```diff
@@ -299,11 +299,7 @@ public function createPlan(){
 
         $this->setRequestOptions();
 
-        $this->response = $this->client->post($this->baseUrl .'/plan', [
-            'body' => json_encode($data)
-        ]);
-
-        return $this->response->getResponse();
+        return $this->setGetResponse("/plan", $data)->getResponse();
```

Comment:
`setGetResponse` method uses the `get` method. How come you are using it here?...shouldn't this be a post request?